chef/cookbooks/hyperv/attributes/default.rb

```diff
@@ -53,24 +53,29 @@
 default[:openstack][:location] = 'C:\OpenStack'
 
 default[:openstack][:nova][:name] = "nova"
-default[:openstack][:nova][:version] = "2014.2.1"
-default[:openstack][:nova][:file] = "nova-2014.2.1.tar.gz"
+default[:openstack][:nova][:version] = "12.0.0"
+default[:openstack][:nova][:file] = "nova-12.0.0.tar.gz"
 default[:openstack][:nova][:target] = "#{node[:openstack][:location]}"
-default[:openstack][:nova][:installed] = "#{node[:python][:path]}\\Scripts\\nova-compute.exe"
+default[:openstack][:nova][:installed] = "#{node[:python][:scripts]}\\nova-compute.exe"
 
 default[:openstack][:neutron][:name] = "neutron"
-default[:openstack][:neutron][:version] = "2014.2.1"
-default[:openstack][:neutron][:file] = "neutron-2014.2.1.tar.gz"
+default[:openstack][:neutron][:version] = "7.0.0"
+default[:openstack][:neutron][:file] = "neutron-7.0.0.tar.gz"
 default[:openstack][:neutron][:target] = "#{node[:openstack][:location]}"
-default[:openstack][:neutron][:installed] = "#{node[:python][:path]}\\Scripts\\neutron-hyperv-agent.exe"
+default[:openstack][:neutron][:installed] = "#{node[:python][:scripts]}\\neutron-hyperv-agent.exe"
+
+default[:openstack][:networking_hyperv][:name] = "networking-hyperv"
+default[:openstack][:networking_hyperv][:version] = "2015.1.1.dev36"
+default[:openstack][:networking_hyperv][:file] = "networking-hyperv-stable-liberty.tar.gz"
+default[:openstack][:networking_hyperv][:target] = "#{node[:openstack][:location]}"
 
 default[:openstack][:ceilometer][:name] = "ceilometer"
-default[:openstack][:ceilometer][:version] = "2014.2.1"
-default[:openstack][:ceilometer][:file] = "ceilometer-2014.2.1.tar.gz"
+default[:openstack][:ceilometer][:version] = "5.0.0"
+default[:openstack][:ceilometer][:file] = "ceilometer-5.0.0.tar.gz"
 default[:openstack][:ceilometer][:target] = "#{node[:openstack][:location]}"
-default[:openstack][:ceilometer][:installed] = "#{node[:python][:path]}\\Scripts\\ceilometer-agent-compute.exe"
 default[:openstack][:ceilometer][:lock_path] = "C:\\OpenStack\\var\\run\\"
 default[:openstack][:ceilometer][:signing_dir] = "C:\\OpenStack\\var\\cache\\ceilometer\\keystone-signing\\"
+default[:openstack][:ceilometer][:installed] = "#{node[:python][:scripts]}\\ceilometer-agent-compute.exe"
```

Comment:
Metrics/LineLength: Line is too long. [105/100] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)